debian/kolibri-server.service

```diff
@@ -10,12 +10,7 @@ Wants=network-online.target
 Type=forking
 ExecStart=/etc/init.d/kolibri-server start
 ExecStop=/etc/init.d/kolibri-server stop
-TimeoutStartSec=infinity
-TimeoutStopSec=10
```

Comment:
Wasn't this timeout supposed to have guaranteed stopping within 10 seconds?

Reply:
Anyways, I agree that we can use default system values and see how that goes... if Stop takes over 10 seconds, I want to know and not have released forced killing of the system process in-the-wild :)

Reply:
Please, test it , four eyes will see more than two. This is one of the changes I tested in:
-  RPi 4
-  a Virtualbox vm
- my own laptop (good cpu with 16 Gb of RAM)

If you can add more scenarios, much better. 
Also I took a look at several Debian packages managing daemons and none of them were using this Timeout options. It seems the defaults usually work